### ISSUE TITLE
Config class for loss functions from MONAI

### ISSUE THREAD

**Issue by thibaultdvx** [aramis-lab/clinicadl#907]
PyTorch's loss functions are quite limited. Even if MONAI's loss function are supported currently, they don't have associated config classes, which limits reproducibility.
